webapp/src/Controller/Jury/ContestController.php

```diff
@@ -855,4 +855,65 @@
             'form' => $form->createView(),
         ]);
     }
+
+    /**
+     * @Route("/{contestId<\d+>}/request-remaining", name="jury_contest_request_remaining")
+     */
+    public function requestRemainingRunsWholeContest(int $contestId): RedirectResponse
+    {
+        /** @var Contest $contest */
+        $contest = $this->em->getRepository(Contest::class)->find($contestId);
+        if (!$contest) {
+            throw new NotFoundHttpException(sprintf('Contest with ID %s not found', $contestId));
+        }
+        $judgings = [];
+        foreach ($contest->getProblems() as $problem) { // This should get the ContestProblem
+            foreach ($problem->getSubmissions() as $submission) { // So this has only submissions in this contest
+                if (!$submission->getTeam()->getCategory()->getVisible()) { // Ignore teams not on the scoreboard
+                    continue;
+                }
+                foreach ($submission->getJudgings() as $judging) { // So these judgings are in scope
+                    if ($judging === null) {
```

Comment:
When does this happen? And do we also want non active judgings (i.e. old ones for rejudgings)? Since that is what you are doing now right?

Reply:
It should never happen, I'll replace it with the check for valid judgings.